src/public/meeting.ts

```diff
@@ -254,6 +254,8 @@ export namespace meeting {
   }
 
   /**
+   * @private
```

Comment:
@aeunms and @charpercyr - when we merge this into 2.0-preview, please make sure to change @private to @hidden for TSDoc. Thanks!